source/common/runtime/runtime_features.cc

```diff
@@ -115,6 +115,8 @@ constexpr const char* disabled_runtime_features[] = {
     "envoy.test_only.per_stream_buffer_accounting",
     // Allows the use of ExtensionWithMatcher to wrap a HTTP filter with a match tree.
     "envoy.reloadable_features.experimental_matching_api",
+    // Use shared cache in getOrCreateRawAsyncClient method.
+    "envoy.reloadable_features.enable_grpc_async_client_cache",
```

Comment:
I think we also need to doc this runtime setting in the release notes, maybe elsewhere also.

Check to see where the other runtime settings are doced.